### ISSUE TITLE
Be able to refer to Regsitry buckets/flows by name

### ISSUE THREAD

**Issue by ghost** [bbende/nifi#51]
It would be nice for things like pg-import to be able to refer to bucket/flow names rather than the Id (which can only be discovered by things like list-buckets). It would make it easier to automate the configuration of Nifi integration with the Registry using tools like Ansible, where it can be hard to lookup Ids.

**Comment by bbende** [comment]
I agree about this idea, unfortunately this requires changes to the REST APIs of NiFi and Registry so it is not something that can be done for the initial version of the tool.

One part of the work would be on registry side:
https://issues.apache.org/jira/browse/NIFIREG-136

After that there would need to be changes on NiFi side to allow the import to use names.
